website/docs/framework-docs/guides/config/configuration.md

```diff
@@ -456,6 +458,10 @@ You can use Environment variables for `user_name` and `password` by using `${env
 When you use Boyka command line assistant to configure your test to run on any Cloud providers using `boyka config [web | api | android | ios] [config_name]` command, then it is **MANDATORY** to provide Environment variable names when prompted for user name and password
 :::
 
+:::warning
+Browser options and Experimental options is not supported for **Safari browser**.
+:::
```

Comment:
Add the following text here:
"Please raise a new feature request alongwith sample example test and options support that is required".